cellengine/utils/api_client/BaseAPIClient.py

```diff
@@ -118,7 +118,9 @@ def _delete(
                 return response.content
             else:
                 raise APIError(
-                    response.url, response.status_code, response["error"]["message"]
+                    response.url,
+                    response.status_code,
+                    response.json() or response["error"],
```

Comment:
`repsonse["error"]` is the HTTP status text for the status code, not the response body property named "error", right?

Reply:
Yes, that's correct; I realized that here I was occasionally returning None instead of the response message